templates/tfengine/recipes/org/project.yaml

```diff
@@ -19,13 +19,18 @@ templates:
   {{else}}
   output_path: "./project.{{.PROJECT_ID}}"
   {{end}}
-- name: "terragrunt"
-  component_path: "../../components/terragrunt/leaf"
+- name: "terraform"
+  recipe_path: "../terraform/terraform.yaml"
   output_ref: "dir"
   output_path: "./project"
+  flatten: [
+    {{if has . "PROJECT_TERRAFORM"}}
```

Comment:
Are you planning to put other things under `PROJECT_TERRAFORM`?

Reply:
PROJECT_TERRAFORM will support any field in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/templates/tfengine/recipes/terraform/terraform.yaml

Reply:
Is it specific to project? Maybe PROJECT_TERRAFORM_ADDONS?

Reply:
Updated, how does it look now?

Reply:
lg